.docker/browser/translations/messages+intl-icu.en.yaml

```diff
@@ -0,0 +1,2 @@
+libero.page:
+    site_name: The International Journal for Demonstration
```

Comment:
Though https://github.com/libero/libero/issues/98 actually has 'The International Journal of Publishing Innovation'. /cc @BlueReZZ

Reply:
That's even more like a potentially real journal name. Go with `The International Journal for Demonstration` and we should raise the difference on https://github.com/libero/libero/issues/98